src/Datadog.Trace/Vendors/Serilog/Core/Logger.cs

```diff
@@ -365,7 +365,7 @@ public void Write<T>(LogEventLevel level, Exception exception, string messageTem
         /// <param name="messageTemplate">Message template describing the event.</param>
         /// <param name="propertyValues">Objects positionally formatted into the message template.</param>
         [MessageTemplateFormatMethod("messageTemplate")]
-        public void Write(LogEventLevel level, Exception exception, string messageTemplate, params object[] propertyValues)
+        public virtual void Write(LogEventLevel level, Exception exception, string messageTemplate, params object[] propertyValues)
```

Comment:
These transforms are applied in the UpdateVendors tool, and they allow us to override the logger behavior through a single method.